metadata/nz_building_outlines.xml

```diff
@@ -104,48 +104,17 @@
-**The following rules were used for capturing buildings;**
-• Buildings under construction are not captured.
-Standalone dwellings:
-• Primary building structures are captured as separate polygons. Adjoining buildings are captured as a single polygon.
-• Extensions, sunrooms, balconies, patios, and annexes are captured as part of the primary building structure.
-• Permanent buildings structures such as sheds and greenhouses, not attached to the primary building structure, but larger than 10m² are captured as individual buildings.
-• Inner courtyards within buildings are captured as a hole within the building polygon if resolution of imagery allows. These holes should be no smaller than 10m².
-Townhouses:
-• Adjoining townhouses are captured a single polygon.
-Commercial:
-• Adjoining commercial buildings are captured as separate polygons with coincident shared edges.
-Rural/farmland areas:
-• Hay bales and silage are not captured.
-• Water tanks are captured as buildings. Minimum size is 16.5m².
-General:
-• Non-permanent objects such as shipping containers, caravans, large vehicles and trampolines should not be captured as buildings.
+This dataset represents the most recent set of building outlines which have been determined from the most recent LINZ aerial imagery available in each region of the country and available on the LINZ Data Service. Current coverage includes areas in Northland, Auckland, Waikato, Hawke’s Bay, Manawatu-Whanganui, Wellington, Tasman, West Coast, Marlborough, Canterbury, Southland and Otago. As new aerial imagery becomes available, this dataset will be updated with new building outlines derived from this new imagery. Since aerial imagery is flown only every few years in each region of the country, building outlines only reflect those buildings observed at the date of this aerial imagery capture, and may not reflect buildings constructed after this time period.  
 
-**Spatial Accuracy:**
-Buildings must be;
-• Spatially accurate to 1 metre
-• Captured in full where they are partially occluded by vegetation or require additional viewer interpretation.
-• Squared off (90 degrees) if the corner angles are between 80-100 degrees.      
+Please refer to the [NZ Building Outlines Data Dictionary] (https://nz-buildings.readthedocs.io/en/latest/introduction.html) for detailed metadata and information about this dataset.
 
-**Valid Geometry:**    
-A building outline polygon is considered to have valid geometry if;
-• It does not overlap with other building outline polygons;
-• It does not contain any spikes;
-• It does not contain lines that are intersected with each other;
-• It can contain polygons with interior rings (holes);
-• It does not contain polygons with multiple exterior rings;
-• Building polygons captured will be greater than 10m².</gco:CharacterString>
+This NZ Building Outlines dataset represents only building outlines derived from the latest aerial imagery in each region. 
```

Comment:
Heading here too and text simplification:

**More building outlines data**
[NZ Building Outlines Full History] (https://data.linz.govt.nz/layer/xxxxx) - contains all combinations of building outlines from multiple years of imagery that have existed since the beginning of this dataset, and the dates when each building outline existed in the associated aerial imagery.

[NZ Building Outlines Lifecycle] (https://data.linz.govt.nz/table/xxxxx) -  is related to the dataset above and stores the relationship between building outlines when one building is split into two or more building outlines, or when multiple building outlines are merged into a single outline.